README.md

```diff
@@ -1,18 +1,83 @@
+- `labelField`: The field in the `soql` query that returns a human-readable label for the experience.
+- `idField`: The field in the `soql` query that returns a unique ID value for the experience.
+- `userSoql`: An SOQL query that requests information about a specific user, by their account ID, with returned values that match the `labelField` and `idField` values. This should include the string `:AccountId`, which will be replaced with the actual user's account ID.
+- `userLabelField`: The field in the `userSoql` query that returns a value to be used as a label for the user's experience. The resulting value would likely match that from `labelField`.
+- `userIdField`: The field in the `userSoql` query that returns an ID that matches one from the general query's `idField`. This is used to match the individual account with one of the available "experiences."
+
+You may also customize the URL that visitors will use to initialize the Salesforce SSO login. Configure the `loginUrl` property for this. By default it will use `/auth/saml/login`. Similarly, you may customize the URL that Salesforce will post the SAML login results to. Set the `callbackUrl` for this, otherwise it defaults to `/auth/saml/login/callback`. You will need to configure your Salesforce connected app with this callback URL.
+
+Example querying [Salesforce Chatter group](https://help.salesforce.com/articleView?id=user_groups.htm&type=5) membership:
+```
+module.exports = {
+  modules: {
+    'apostrophe-salesforce-experience': {
+      adminUsername: 'admin@organization.org',
+      adminPasswordAndToken: 'password0a1b2c3d4e5f',
+      adminLoginUrl: 'https://example.my.salesforce.com/',
+      redirectionUrl: 'https://example.lightning.force.com/lightning/page/home',
+      experienceQueries: [
+        {
+          // General query and field names
+          soql: 'SELECT Name, Id FROM CollaborationGroup',
+          labelField: 'Name',
+          idField: 'Id',
+          // User query and field names
+          userSoql: 'SELECT Name, Id FROM CollaborationGroup WHERE Id IN (SELECT CollaborationGroupId FROM CollaborationGroupMember WHERE MemberId = :AccountId)',
+          userLabelField: 'Name',
+          userIdField: 'Id'
+        }
+      ]
+      // loginUrl: '/another/login', // Optional
+      // callbackUrl: '/another/callback' // Optional
+    }
+  }
+};
+```
+
+### User flow
+
+Once configured, a Salesforce account holder **with access through the connected app you created for the SAML connection** can visit the login url (`loginUrl`), which should send them to the Salesforce login page. Once they successfully sign in, they will be redirected back to the website, or to whatever `redirectionUrl` you set. They should then be logged into both Salesforce and be recognized by the Apostrophe applications. By visiting the `/auth/saml/logout` URL of the Apostrophe application they will be disconnected from the app as well as logged out from Salesforce.
```

Comment:
"They should then be logged into both Salesforce and recognized by the Apostrophe applications..."

How about:

"The website will then be able to access their Salesforce profile, and they will also be logged into Salesforce, which is useful if you choose to redirect them there."